src/test/unit/gov/nist/javax/sip/stack/DialogEarlyStateTimeoutWithout100Test.java

```diff
@@ -0,0 +1,479 @@
+package test.unit.gov.nist.javax.sip.stack;
+
+import gov.nist.javax.sip.DialogTimeoutEvent;
+import gov.nist.javax.sip.ServerTransactionExt;
+import gov.nist.javax.sip.SipListenerExt;
+import gov.nist.javax.sip.SipProviderExt;
+import gov.nist.javax.sip.SipStackExt;
+import gov.nist.javax.sip.header.HeaderFactoryExt;
+import gov.nist.javax.sip.message.MessageFactoryExt;
+import gov.nist.javax.sip.stack.NioMessageProcessorFactory;
+
+import java.util.ArrayList;
+import java.util.Properties;
+
+import javax.sip.ClientTransaction;
+import javax.sip.Dialog;
+import javax.sip.DialogTerminatedEvent;
+import javax.sip.IOExceptionEvent;
+import javax.sip.ListeningPoint;
+import javax.sip.PeerUnavailableException;
+import javax.sip.RequestEvent;
+import javax.sip.ResponseEvent;
+import javax.sip.ServerTransaction;
+import javax.sip.SipFactory;
+import javax.sip.SipListener;
+import javax.sip.TimeoutEvent;
+import javax.sip.TransactionState;
+import javax.sip.TransactionTerminatedEvent;
+import javax.sip.address.Address;
+import javax.sip.address.AddressFactory;
+import javax.sip.address.SipURI;
+import javax.sip.header.CSeqHeader;
+import javax.sip.header.CallIdHeader;
+import javax.sip.header.ContactHeader;
+import javax.sip.header.ContentTypeHeader;
+import javax.sip.header.ExpiresHeader;
+import javax.sip.header.FromHeader;
+import javax.sip.header.MaxForwardsHeader;
+import javax.sip.header.RouteHeader;
+import javax.sip.header.ToHeader;
+import javax.sip.header.ViaHeader;
+import javax.sip.message.Request;
+import javax.sip.message.Response;
+
+import org.apache.log4j.ConsoleAppender;
+import org.apache.log4j.Logger;
+
+import test.unit.gov.nist.javax.sip.stack.CtxExpiredTest.Shootist;
+import test.unit.gov.nist.javax.sip.stack.CtxExpiredTest.Shootme;
+import junit.framework.TestCase;
+
+public class DialogEarlyStateTimeoutWithout100Test extends TestCase {
```

Comment:
Should we merge this test class with the "default" one since they use most of the same utilities and are tightly related ?

Reply:
Reworked the test, making it different from the test is is copied off.

Reply:
Like the new tests! 👍